apps/web/composables/useModernImage/__tests__/useModernImage.spec.ts

```diff
@@ -0,0 +1,123 @@
+// write a test for the addModernImageExtension function
+
+import { mockNuxtImport } from "@nuxt/test-utils/runtime";
+import { useModernImage } from "../useModernImage";
+
+describe('useModernImage', () => {
+
+    const { useRuntimeConfig } = vi.hoisted(() => {
+        return {
+            useRuntimeConfig: vi.fn().mockImplementation(() => {
+                return {
+                    public: {
+                        useAvif: true,
+                        useWebp: true
+                    }
+                }
+            })
+        }
+    });
+
+    mockNuxtImport('useRuntimeConfig', () => {
+        return useRuntimeConfig
+    });
+
+    afterEach(() => {
+        vi.clearAllMocks();
+        useRuntimeConfig.mockImplementation(() => {
+            return {
+                public: {
+                    useAvif: true,
+                    useWebp: true
+                }
+            }
+        });
```

Comment:
**question**: What's the point of creating a mock during the cleanup step? It seems like that shouldn't do anything.